### PR TITLE
feat: replace StaticMap By Maplibre

### PR DESCRIPTION
**Pull request by unclFedor**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,14 +39,15 @@
     "@math.gl/proj4": "^3.6.3",
     "@probe.gl/stats": "^4.0.4",
     "@reduxjs/toolkit": "^1.9.5",
-    "deck.gl": "^8.8.3",
+    "apache-arrow": "^13.0.0",
```
Comment (belom88) [inline package.json:42]: Is it required. We don't have table formats in this application

File: src/components/deck-gl-wrapper/deck-gl-wrapper.tsx [hunk 2]
```diff
@@ -761,21 +767,36 @@ export const DeckGlWrapper = ({
...
-          preventStyleDiffing
-          preserveDrawingBuffer
-        />
+          terrain={
+            showTerrain
+              ? { source: "dem-data-source", exaggeration: 1 }
+              : undefined
+          }
+        >
+          <MaplibreSource
```
Comment (belom88) [inline src/components/deck-gl-wrapper/deck-gl-wrapper.tsx:778]: We don't need maplibre terrain now

File: src/pages/dashboard/dashboard.tsx [hunk 3]
```diff
@@ -322,7 +325,18 @@ export const Dashboard = () => {
           layers={[tile3DLayer]}
           initialViewState={viewState}
         >
-          <StaticMap mapStyle={DEFAULT_MAP_STYLE} />
+          <MaplibreMap mapStyle={DEFAULT_MAP_STYLE}>
+            <MaplibreSource
```
Comment (belom88) [inline src/pages/dashboard/dashboard.tsx:329]: Remove maplibre terrain

**Review by belom88 — CHANGES_REQUESTED** [review]

**Review by belom88 — CHANGES_REQUESTED** [review]
What hapened with `yarn.lock`? Do you use `npm` instead of `yarn`?

File: package.json (hunk 1 above)
Comment (unclFedor) [reply package.json:42]: Yes, there is compilation error without "apache-arrow": "^13.0.0" when I replaced "deck.gl": "^8.8.3" by list of modules   "@deck.gl/XXXX": "^8.9.32",

File: src/components/deck-gl-wrapper/deck-gl-wrapper.tsx (hunk 2 above)
Comment (unclFedor) [reply src/components/deck-gl-wrapper/deck-gl-wrapper.tsx:778]: removed condition and kept terrain={undefined}

File: src/pages/dashboard/dashboard.tsx (hunk 3 above)
Comment (unclFedor) [reply src/pages/dashboard/dashboard.tsx:329]: Removed  <MaplibreSource>

**Comment by unclFedor** [comment]
> What hapened with `yarn.lock`? Do you use `npm` instead of `yarn`?

I restored previous version of yarn.lock and made changes again by using yarn add/remove.

**Review by belom88 — CHANGES_REQUESTED** [review]

File: src/redux/slices/colors-by-attribute-slice.ts [hunk 4]
```diff
@@ -2,6 +2,10 @@ import { PayloadAction, createSlice } from "@reduxjs/toolkit";
 import { ColorsByAttribute } from "../../types";
 import { RootState } from "../store";
 
+jest.mock("@loaders.gl/i3s", () => ({
```
[the remark this answers is not in the document]
Comment (unclFedor) [reply src/redux/slices/colors-by-attribute-slice.ts]: removed jest.mock from slice

File: src/components/comparison/comparison-side/comparison-side.spec.tsx [hunk 5]
```diff
@@ -18,6 +18,21 @@ import { MemoryUsagePanel } from "../../memory-usage-panel/memory-usage-panel";
 import { setupStore } from "../../../redux/store";
 import { setDragMode } from "../../../redux/slices/drag-mode-slice";
 
+jest.mock("@deck.gl/geo-layers", () => ({
+  load: jest.fn(),
+}));
```
[the remark this answers is not in the document]
Comment (unclFedor) [reply src/components/comparison/comparison-side/comparison-side.spec.tsx]: removed load from jest.mock

File: src/components/deck-gl-wrapper/deck-gl-wrapper.tsx [hunk 6]
```diff
@@ -761,21 +764,11 @@ export const DeckGlWrapper = ({
         currentViewport = viewport;
       }}
       {!showTerrain && (
-        <StaticMap
-          reuseMaps
-          mapStyle={mapStyle}
-          preventStyleDiffing
-          preserveDrawingBuffer
-        />
+        <MaplibreMap mapStyle={mapStyle} terrain={undefined}></MaplibreMap>
```
Comment (belom88) [inline src/components/deck-gl-wrapper/deck-gl-wrapper.tsx:767]:
```suggestion
        <MaplibreMap mapStyle={mapStyle}></MaplibreMap>
```

**Review by belom88 — APPROVED** [review]

**Pull request merged**